ambry-protocol/src/test/java/com.github.ambry.protocol/RequestResponseTest.java

```diff
@@ -268,15 +274,19 @@ public void getRequestResponseTest() throws IOException {
     Assert.assertEquals(deserializedGetResponse.getError(), ServerErrorCode.No_Error);
     Assert.assertEquals(deserializedGetResponse.getPartitionResponseInfoList().size(), 1);
     Assert.assertEquals(deserializedGetResponse.getPartitionResponseInfoList().get(0).getMessageInfoList().size(), 1);
-    Assert.assertEquals(
-        deserializedGetResponse.getPartitionResponseInfoList().get(0).getMessageInfoList().get(0).getSize(), 1000);
-    Assert.assertEquals(
-        deserializedGetResponse.getPartitionResponseInfoList().get(0).getMessageInfoList().get(0).getStoreKey(), id1);
-    Assert.assertEquals(deserializedGetResponse.getPartitionResponseInfoList()
-        .get(0)
-        .getMessageInfoList()
-        .get(0)
-        .getExpirationTimeInMs(), 1000);
+    MessageInfo msgInfo = deserializedGetResponse.getPartitionResponseInfoList().get(0).getMessageInfoList().get(0);
+    Assert.assertEquals(msgInfo.getSize(), 1000);
+    Assert.assertEquals(msgInfo.getStoreKey(), id1);
+    Assert.assertEquals(msgInfo.getExpirationTimeInMs(), 1000);
+    if (GetResponse.getCurrentVersion() == GetResponse.Get_Response_Version_V3) {
```

Comment:
not for now. But have tested this by switching current version to 3 in prod class. Don't need to do anything when we make the switch to V3